data/json/morale_types.json

```diff
@@ -324,6 +324,11 @@
     "type": "morale_type",
     "text": "Anguished by memories of butchering human corpse"
   },
+  {
+    "id": "morale_pain",
+    "type": "morale_type",
+    "text": "In pain"
```

Comment:
This was committed accidentally, I assume.

Reply:
dammit, I am on fire tonight with these mistakes. ( hah )